packages/web-frontend/src/routes/+layout.svelte

```diff
@@ -1,6 +1,7 @@
 <script>
 	import Header from './Header.svelte';
 	import './styles.css';
+	import "../app.css";
```

Comment:
Why is this needed?

Reply:
This was part of the tailwind documentation for integrating with svelte.